scripts/generate_access_token.sh

```diff
@@ -66,7 +66,7 @@ if [[ $GHA_SECRET == "local_dev" ]]; then
     cp $SERVICE_ACCOUNT ~/.credentials/
 else
     echo "Decrypting ${SERVICE_ACCOUNT_FILE}.gpg"
-    scripts/decrypt_GHA_SECRET.sh $SERVICE_ACCOUNT ~/.credentials/$SERVICE_ACCOUNT_FILE "$plist_secret"
+    scripts/decrypt_gha_secret.sh $SERVICE_ACCOUNT ~/.credentials/$SERVICE_ACCOUNT_FILE "$GHA_SECRET"
```

Comment:
This shouldn't fix the nightly test errors seen in #9086 but are improvements I made while trying to reproduce nightly failures.

Reply:
Assuming that `"$plist_secret"` and `"$GHA_SECRET"` are equivalent?

Reply:
Yes. `plist_secret` is the env var that GHA knows about. We pass it to the script like so:

https://github.com/firebase/firebase-ios-sdk/blob/0e855f4ef6dffe30e2e9375b6034d5d76e5345f4/.github/workflows/remoteconfig.yml#L35

After some head scratching while running locally, I realized that I was using the expected GHA env var in the diff'd code rather than the local param.